ports/rapidjson/vcpkg.json

```
@@ -1,9 +1,9 @@
 {
   "name": "rapidjson",
-  "version-date": "2020-09-14",
-  "port-version": 2,
+  "version-date": "2021-11-01",
```

```suggestion
  "version-date": "2022-6-28",
```
Please run the command `./vcpkg x-add-version rapidjson` after modifying the `version-date` field